app/src/main/java/com/provider/service/EntityConverterService.java

```diff
@@ -65,4 +71,13 @@ public void patchRequestModelToProvider (ProviderUpdateRequestModel providerUpda
         modelMapper.map(providerUpdateRequestModel, provider);
     }
 
+    public void patchItemUpdateModelToItem (ItemUpdateRequestModel itemUpdateRequestModel, Item item) {
+        modelMapper.map(itemUpdateRequestModel, item);
```

Comment:
Reminder: this shouldn’t be patch.

And of course this will not work with null values.